web/styles/dark_theme.css

```diff
@@ -172,7 +172,7 @@
     }
 
     & div.overlay .flex.overlay-content > .overlay-container,
-    #settings_page,
```

Comment:
I guess this was removed by mistake, since this class is not used for streams UI and checking out to this commit breaks the design for settings overlay in dark theme.